texar/tf/modules/decoders/transformer_decoders.py

```diff
@@ -319,9 +319,9 @@ def _build(self,  # pylint: disable=arguments-differ, too-many-statements
         """Performs decoding.
 
         The interface is mostly the same with that of RNN decoders
-        (see :meth:`~texar.tf.modules.RNNDecoderBase._build`). The main difference
-        is that, here, `sequence_length` is not needed, and continuation
-        generation is additionally supported.
+        (see :meth:`~texar.tf.modules.RNNDecoderBase._build`). The main
+        difference is that, here, `sequence_length` is not needed, and
+        continuation generation is additionally supported.
```

Comment:
Pls update the docstring to reflect this request: https://github.com/asyml/texar/issues/249#issuecomment-557249347 ,  and close the issue when merging the PR